ros2pkg/ros2pkg/verb/create.py

```diff
@@ -89,6 +89,21 @@ def add_arguments(self, parser, cli_name):
             help='name of the empty library')
 
     def main(self, *, args):
+
+        def get_git_config(key: str) -> str | None:
```

Comment:
```suggestion
        def get_git_config(key: str) -> Optional[str]:
```

Make sure to import it:

```
from typing import Optional
```